tests/test_text/test_time_intervals.py

```diff
@@ -96,3 +97,40 @@ def test_csv_round_trip_rename(tmp_path):
 def test_get_metadata_schema():
     interface = CsvTimeIntervalsInterface(trials_csv_path)
     interface.get_metadata_schema()
+
+
+def test_trials():
+    metadata = dict(
+        NWBFile=dict(
+            session_start_time=datetime.now().astimezone(),
+        ),
+        TimeIntervals=dict(
+            trials=dict(
+                columns=dict(
+                    start_time=dict(description="start time of the trial"),
+                    stop_time=dict(description="stop time of the trial"),
+                    correct=dict(description="correct or not"),
```

Comment:
I was expecting that you had to pass the name inside the dictionary as well from reading the schema, but here, the name here is only passed as the dictionary key.